### DISCUSSION TITLE
Finite State Machine lacks of documentation

### DISCUSSION THREAD

**Discussion by Sareff** [aiogram/aiogram#888] [Q&A] [answered]
Description:
While I was working with Aiogram, I've stumbled upon a serious issue - there is no a documentation for States in Finite State Machine (https://docs.aiogram.dev/en/latest/dispatcher/fsm.html?highlight=state#state).
This single fact ruins my participation in Hackathon, because States was developed at the last moment.

Solution:
Write it down.

**Comment by JrooTJunior** [comment, marked as answer]
2.x version has much many examples in repository and repositories on github and guides but the documentation for 2.x versions will no more be updated because we have 3.0 in beta stage with documentation
